types/semantic-release/index.d.ts

```diff
@@ -0,0 +1,46 @@
+// Type definitions for semantic-release 15.13
+// Project: https://github.com/semantic-release/semantic-release#readme
+// Definitions by: Leonardo Gatica <https://github.com/lgaticaq>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * The semantic release configuration itself.
+ */
+export interface GlobalConfig {
```

Comment:
This is fine for this PR, but in future PRs adding types you'll have to use [this template](https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-function-d-ts.html) since the underlying library is exporting a function using CommonJS convention. Likewise you'll need to use the [require syntax](https://www.typescriptlang.org/docs/handbook/modules.html#export--and-import--require) in your tests to import it.